openedx_lti_tool_plugin/views.py

```diff
@@ -55,11 +64,28 @@ def setup(self, request: HttpRequest, *args: tuple, **kwargs: dict):
             request: HTTP request object.
             *args: Variable length argument list.
             **kwargs: Arbitrary keyword arguments.
+
         """
         super().setup(request, *args, **kwargs)
         self.tool_config = DjangoDbToolConf()
         self.tool_storage = DjangoCacheDataStorage(cache_name='default')
 
+    def http_response_error(self, message: Union[str, Exception]) -> LoggedHttpResponseBadRequest:
+        """HTTP response with an error message.
+
+        This method will create a HTTP response error with an error message
+        prefixed with the LTI specification version and the view name of the error.
+
+        Args:
+            message: Error message string or Exception object.
+
+        Returns:
+            LoggedHttpResponseBadRequest object with error message
+                prefixed with LTI version and view name.
+
+        """
+        return LoggedHttpResponseBadRequest(f'LTI 1.3 {self.__class__.__name__}: {message}')
+
```

Comment:
What do you think of moving this to it's own commit, separate from deep linking implementation?

Reply:
@alexjmpb I think it's not necessary to separate this on a commit in this PR, and I think opening a PR for this change will be a bigger effort than just including this change on this PR